CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Change log
 
+## 2.0.5 (2023-02-13)
+- fix: resolve the error that occurs when encrypting an email through AWS SES using KMS, then decrypting it using this library.
```

Comment:
Can we say something like that gives some context on what an email and S3 have anything to do with each other?

```
Fixed issue when AWS SES is configured to send encrypted emails to S3 bucket and then fail to be decrypted by Amazon.Extensions.S3.Encryption.
```

Reply:
Yup I agree thanks! Let me change it.